paradise_papers_django/requirement.txt

```diff
@@ -0,0 +1,5 @@
+django-neomodel==0.0.4
```

Comment:
We need a requirement folder with the requirement file for each enviroment
```
     requirements/
         prod.txt
         local.txt
```
And all the setting necessaries to automatize this process